src/commands/autoremove.cc

```diff
@@ -0,0 +1,121 @@
+/*---------------------------------------------------------------------------*\
+                          ____  _ _ __ _ __  ___ _ _
+                         |_ / || | '_ \ '_ \/ -_) '_|
+                         /__|\_, | .__/ .__/\___|_|
+                             |__/|_|  |_|
+\*---------------------------------------------------------------------------*/
+#include <zypp/ZYpp.h> // for ResPool::instance()
+#include "autoremove.h"
+#include "utils/flags/flagtypes.h"
+#include "commands/conditions.h"
+#include "solve-commit.h"
+#include "utils/messages.h"
+#include "src/SolverRequester.h"
+#include "commonflags.h"
+#include "Zypper.h"
+
+extern ZYpp::Ptr God;
+
+AutoRemoveCmd::AutoRemoveCmd(std::vector<std::string> &&commandAliases_r) : ZypperBaseCommand(std::move(commandAliases_r),
+                                                                                              // translators: command synopsis; do not translate lowercase words
+                                                                                              _("autoremove (arm) [OPTIONS]"),
+                                                                                              // translators: command summary: dist-upgrade, dup
```

Comment:
Copy&paste? "command summary: autoremove, arm"?

Reply:
ah copy and past my bad